modules/ROOT/partials/misc/admon-ai-proxy.adoc

```diff
@@ -0,0 +1 @@
+IMPORTANT: The following example stores the authentication credentials in the client side integration. This is not recommended for any purpose. Instead, an alternate method for retrieving the credentials should be used such as a xref:ai-proxy.adoc[proxy server] or a server-side integration.
```

Comment:
```suggestion
IMPORTANT: For the sake of demonstration, the following examples show how to use the authentication credentials  with the API within the client side integration. This is not recommended for production purposes. We recommend only accessing the API with a xref:ai-proxy.adoc[proxy server] or implementing a server-side integration. This will prevent unauthorized access to the API.
```
Just that we are recommending that all credentials are hidden, as they shouldn't be retrieved by the client side.